pythran/pythonic/include/types/tuple.hpp

```diff
@@ -1115,23 +1115,32 @@ namespace sutils
   using push_front_t = concat_t<types::pshape<T>, P>;
 
   template <class S>
-  long find(S &s, long v, std::integral_constant<size_t, 0>)
+  long find(S &s, long v, std::integral_constant<size_t, 0>, long start,
+            std::function<bool(long, long)> comp)
```

Comment:
instead of a std::function, just use a template parameter, that way we avoid the std::function overhead.